cmd/podman/umount.go

```diff
@@ -16,13 +16,23 @@ var (
 			Name:  "all, a",
 			Usage: "umount all of the currently mounted containers",
 		},
+		cli.BoolFlag{
+			Name:  "force, f",
+			Usage: "force the complete umount all of the currently mounted containers",
+		},
 	}
 
+	description = `
+Container storage keeps track of the number of times the container was mounted.
```

Comment:
Suggest rewording: "Decrement the mount counter for a container, unmounting the container if the counter reaches 0. Unmounting can be force with the --force flag."